data/gui/window/migrate_version_selection.cfg

```diff
@@ -0,0 +1,122 @@
+#textdomain wesnoth-lib
+###
+### Definition of the window to ask which Wesnoth version to migrate data from
+###
+
+[window]
+	id = "migrate_version_selection"
+	description = "Dialog to select a previous version of Wesnoth to migrate data from."
+
+	[resolution]
+		definition = "default"
+
+		[linked_group]
+			id = "version_group"
+			fixed_width = true
+		[/linked_group]
+
+		[linked_group]
+			id = "radio_options"
+			fixed_width = true
+		[/linked_group]
+
+		[tooltip]
+			id = "tooltip"
+		[/tooltip]
+
+		[helptip]
+			id = "tooltip"
+		[/helptip]
+
+		[grid]
+			[row]
+				[column]
+
+					[label]
+						definition = "default"
+						label = _ "Choose which version of Wesnoth to import your data from:"
+					[/label]
+
+				[/column]
+			[/row]
+			[row]
+				[column]
+					[listbox]
+						id = "versions_listbox"
+						definition = "default"
+						horizontal_scrollbar_mode = "never"
+
+						[list_definition]
+							[row]
+								[column]
+									vertical_grow = true
+									horizontal_grow = true
+									[toggle_panel]
+										definition = "default"
+										[grid]
+											[row]
+												[column]
+													grow_factor = 1
+													horizontal_alignment = "left"
+
+													[label]
+														id = "version_label"
+														definition = "default"
+													[/label]
+
+												[/column]
+
+												[column]
+													grow_factor = 1
+													horizontal_alignment = "center"
+													[toggle_button]
```

Comment:
There's no reason to have a toggle button here… I'd just remove it altogether. If a radiobutton is desired it could be added to the toggle panel definition.

Reply:
I don't understand - a toggle button is essentially a radio button/checkbox widget isn't it?

Reply:
Yes, but a radio button doesn't make sense in this context. The list itself is your choice. If you mean this to be a checkbox rather than a radio button (to migrate from potentially multiple version), then I suppose that could make sense, though I imagine making that actually work is quite a bit harder.